packages/lib-user/test/mocks/panoptes.mock.js

```diff
@@ -0,0 +1,57 @@
+const PROJECTS = [
```

Comment:
I should probably create a `panoptes` folder with the various resources as individual files. That could be included in this PR.

I should also refactor with `rosie` similar to [lib-classifier](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/test/factories), but that might be for a subsequent PR.